regression/esbmc-unix2/01_pthread14/test.desc

```diff
@@ -1,4 +1,4 @@
 CORE
 main.c
-
+--no-por
```

Comment:
Would it be possible to add a brief comment in this PR to explain why partial-order reduction is not needed? 
just for future reference if someone has the curiosity and traced it back to this PR ...

Reply:
Another good catch. Our partial-order reduction is buggy, but I don't think there is an issue open for that. I'll create one now. Just for reference, we don't use POR in SV-COMP because it is removing valid transitions.